### PR TITLE
Dance page - style playlist thumbnail

### PR DESCRIPTION
**Pull request by nkiruka**
The playlist thumbnail image is updated and styled.

Implementation
- [x] Import new asset
- [x] Apply styles to playlist thumbnail
- [x] Make thumbnail responsive

### Screen shots
- Before
screen shot 2018-11-09 at 2 06 13 pm

- Not responsive
screen shot 2018-11-09 at 2 07 17 pm

- After
screen shot 2018-11-09 at 3 05 32 pm

- Responsive
screen shot 2018-11-09 at 3 06 10 pm

### CODE REVIEW

**Comment by nkiruka** [comment]
@tanyaparker Should the total playlist time be added?

**Comment by nkiruka** [comment]
Resolving conflict

**Comment by nkiruka** [comment]
@tanyaparker @Erin007 Merge conflict resolved

**Comment by Erin007** [comment]
Do we lose the "What is Creativity? View full series" label on mobile or just cut off in the screenshots?

**Comment by nkiruka** [comment]
@Erin007 The label is still visible; I have updated the PR.

File: pegasus/sites.v3/code.org/public/dance.haml [hunk 1]
```diff
@@ -21,11 +22,15 @@ theme: responsive
...
 %h2= I18n.t(:hoc2018_dance_videos_section_heading)
-.dance-videos.dance-clear-element
-  =view :dance_creativity_video
-  %a{href: "https://www.youtube.com/playlist?list=PLzdnOPI1iJNcXMS80NRhQd0NA3po1swJf"}
-    %img.dance_thumbnail-tile-img{src:"/images/creativity_playlist_thumbnail.png"}
-%h5.dance_thumbnail-tile-subtitle= I18n.t(:hoc2018_dance_view_creativity_video_playlist)
+.thumbnail-wrapper
+  .dance-videos.dance-clear-element
+    =view :dance_creativity_video
+    .col-50{style: "padding: 10px"}
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/dance.haml]: pull out your inline styles, please 🙂

Comment (nkiruka) [reply pegasus/sites.v3/code.org/public/dance.haml]: Oops!  thanks

**Comment by tanyaparker** [comment]
image

This is not responsive because the playlist image needs to keep the same dimension ratio. It shouldn't squeeze to fit the same size as the YT thumbnails. It's ok if it's a little smaller or bigger than the YT thumbnails, as long as it maintains its dimension ratio.

**Pull request merged**